lib/inspec/resources/cassandra.rb

```diff
@@ -0,0 +1,64 @@
+module Inspec::Resources
+  class Cassandra < Inspec.resource(1)
+    name "cassandra"
+    supports platform: "unix"
+    supports platform: "windows"
+
+    desc "The 'cassandra' resource is a helper for the 'cql_conf'"
+
+    attr_reader :conf_path
+
+    def initialize
+      case inspec.os[:family]
+      when "debian", "redhat", "linux", "suse"
+        determine_conf_dir_and_path_in_linux
+      when "windows"
+        determine_conf_dir_and_path_in_windows
+      end
+    end
+
+    def to_s
+      "CassandraDB"
+    end
+
+    private
+
+    def determine_conf_dir_and_path_in_linux
+      cassandra_home = inspec.os_env("CASSANDRA_HOME").content
+
+      if cassandra_home.nil? || cassandra_home.empty?
+        warn "$CASSANDRA_HOME env value not set in the system"
+        nil
+      else
+        conf_path = "#{cassandra_home}/cassandra.yaml"
+        if !inspec.file(conf_path).exist?
+          warn "No cassandra conf file found in CASSANDRA_HOME directory"
```

Comment:
We change the test like this "Cassandra conf file not found in CASSANDRA_HOME directory."
Another thing can we get the path from CASSANDRA_HOME and display the path in the warning too so that in the warning itself they will be able to see the path of the directory in which it's trying to locate the file?

Reply:
Should this part need to be in the cassendra_conf resource file?

Reply:
I feel we should put in cassandra file itself because it is making the code clean on conf file side.